src/main/resources/application.yml

```diff
@@ -15,6 +15,8 @@ queueconfig:
   case-event-exchange: events
   refusal-queue: FieldworkAdapter.Refusals
   refusal-routing-key: event.respondent.refusal
+  invalid-address-routing-key: event.case.address.update
+  invalid-address-inbound-queue: FieldworkAdapter.invalidAddress
```

Comment:
Unbelievably tiny comment (feel free to ignore!) but might this be `FieldworkAdapter.InvalidAddress` to match other field queues?